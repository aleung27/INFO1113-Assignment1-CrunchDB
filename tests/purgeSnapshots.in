SET k 9
SNAPSHOT
SET g 8
SNAPSHOT
SET q 0
SNAPSHOT
LIST SNAPSHOTS
PURGE k
CHECKOUT 1
LIST ENTRIES
CHECKOUT 2
LIST ENTRIES
CHECKOUT 3
LIST ENTRIES
BYE
