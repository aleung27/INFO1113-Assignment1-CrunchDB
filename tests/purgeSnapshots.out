> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> 3
2
1

> ok

> ok

> no entries

> ok

> g [8]

> ok

> q [0]
g [8]

> bye
